wpilibj/src/main/java/edu/wpi/first/wpilibj/PIDController.java

```diff
@@ -18,7 +18,10 @@
  * <p>This feedback controller runs in discrete time, so time deltas are not used in the integral
  * and derivative calculations. Therefore, the sample rate affects the controller's behavior for a
  * given set of PID constants.
+ *
+ * @deprecated Use controllers.PIDController class instead.
  */
+@Deprecated
```

Comment:
Add deprecation info

```suggestion
@Deprecated(since = "2020", forRemoval = true)
```

Reply:
Should maybe file an issue to do this everywhere.